neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -184,16 +206,67 @@ def _substitute_helpers(self, derivatives, helpers):
         substitutions = {helper[0]: helper[1] for helper in sympified_helpers}
         return [derivative.subs(substitutions) for derivative in sympified_derivatives]
 
-    def run(self, duration, dt, noise_input, **kwargs):
+    @staticmethod
+    def _get_numba_function_params(symbol_params):
```

Comment:
this function returns a full list of all model parameters that are inserted into the `integrate` function